scripts/build-examples.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-env node */
+'use strict';
+
+const path = require('path');
+const fs = require('fs-extra');
+
+const DEST_DIR = 'dist';
+
+// If invoked via "npm run examples" or "node scripts/build-examples.js" run immediately
+const calledFromCli = !module.parent;
+if (calledFromCli) {
+    buildExamples();
+}
+
+async function getFiles(dir) {
+    const subdirs = await fs.readdir(dir);
+    const files = await Promise.all(subdirs.map(async (subdir) => {
+        const res = path.resolve(dir, subdir);
+        return (await fs.stat(res)).isDirectory() ? getFiles(res) : res;
+    }));
+    return files.reduce((a, f) => a.concat(f), []);
+}
+
+async function buildExamples(destPath = DEST_DIR, sdkUrl = null) {
+    await fs.copy('examples', destPath);
+    const files = await getFiles(destPath);
+    for (let file of files) {
```

Comment:
I dont think we're guaranteed of a clean dist directory here.  there might be other build artifacts that could match and get corrupted.  I wonder if we should use a temp directory to mutate and then copy into dest_dir?  Or, we could look into streaming options?

Reply:
Yeah, that's a fair point. I'll look into this!